backend/src/utils/banner.helper.js

```
@@ -1,16 +1,132 @@
+  static async handleAsync(res, operation, errorType) {
+    try {
+      const result = await operation();
+      return result;
+    } catch (err) {
+      console.error(`${errorType}: `, err);
+      const { statusCode, payload } = internalServerError(errorType, err.message);
+      res.status(statusCode).json(payload);
+      return null;
+    }
+  }
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)